fitapp/migrations/0005_upgrade_oauth1_tokens_to_oauth2.py

```diff
@@ -0,0 +1,44 @@
+import base64
+import requests
+
+from django.db import migrations, models
+
+from fitapp.utils import get_setting
+
+
+def forwards(apps, schema_editor):
+    client_id = get_setting('FITAPP_CONSUMER_KEY')
+    client_secret = get_setting('FITAPP_CONSUMER_SECRET')
+    encoded = base64.b64encode('{0}:{1}'.format(client_id, client_secret))
+    headers = {'Authorization': 'Basic {0}'.format(encoded)}
+
+    request_token_url = 'https://api.fitbit.com/oauth2/token'
+    UserFitbit = apps.get_model('fitapp', 'UserFitbit')
+    for fbuser in UserFitbit.objects.filter(refresh_token=''):
+        data = {
+            'grant_type': 'refresh_token',
+            'refresh_token': '{0}:{1}'.format(
+                fbuser.access_token, fbuser.auth_secret)
+        }
+        resp = requests.post(request_token_url, headers=headers, data=data)
+        if resp.status_code == 200:
+            content = resp.json()
+            fbuser.access_token = content['access_token']
+            fbuser.refresh_token = content['refresh_token']
+            fbuser.save()
```

Comment:
@percyperez I think you can simplify this by using the `refresh_token` method of `FitbitOAuth2Client`. Something like:
```python
token = FitbitOAuth2Client(
    get_setting('FITAPP_CONSUMER_KEY'),
    get_setting('FITAPP_CONSUMER_SECRET'),
    refresh_token = '{0}:{1}'.format(fbuser.access_token, fbuser.auth_secret)
).refresh_token()
fbuser.access_token = token['access_token']
fbuser.refresh_token = token['refresh_token']
fbuser.save()
```